config/telegram.env.dist

```diff
@@ -1,2 +1,8 @@
 TELEGRAM_API_TOKEN=""
 TELEGRAM_ADMIN_ID=""
+MIGRATIONS_PATH="./migrations"
+DB_LOGIN="postgres"
```

Comment:
The names should be consistent - if you have something named `DB_*` you cannot have name `DATABASE_*` and vise versa.